### DISCUSSION TITLE
Platform positioning and 3D Text

### DISCUSSION THREAD

**Discussion by schmitt-steven** [OctoPi-Team/OctoPi#44] [General]
What do y'all think about the platforms positioning in the branch '_platform-positioning_'? 
They are now either higher, lower or on the same level in comparison to the main platform. I think the platform layout  looked a bit boring before. 
Also placed the 3D Texts although Im not sure about their design yet.
![platforms1](https://github.com/OctoPi-Team/OctoPi/assets/82405062/8dbdbc54-7c1d-4630-bf81-8cf1f6225c9d)

**Comment by pIlIp-d** [comment]
Ja, lässt sich noch schwer lesen aber sieht schonmal nice aus. Vlt keine abrundung oder andere farbe etc
